test/platform/azure/index.spec.ts

```diff
@@ -1,6 +1,6 @@
 import is from '@sindresorhus/is';
 import * as _hostRules from '../../../lib/util/host-rules';
-import { InitRepoConfig } from '../../../lib/platform/common';
+import { RepoParams } from '../../../lib/platform/common';
```

Comment:
Maybe we should add `export * from './common'` to `platform/index.ts`, then we can reduce the nested import.

`import { RepoParams } from '../../../lib/platform';`

Reply:
I don't think we import from platform/index though?

Reply:
The `/index` is optional 😄